### ISSUE TITLE
Fix compilation for AtomicFetchOp for ARM

### ISSUE THREAD

**Issue by hernanponcedeleon** [hernanponcedeleon/Dat3M#58]

**Comment by hernanponcedeleon** [comment]
@ThomasHaas can you check the compilation scheme in branch 58 and see if it looks ok?

**Comment by ThomasHaas** [comment]
It looks ok. But we should change to a strong RMWStoreExclusive once we have it. AtomicFetchOps should not fail spuriously.

**Comment by hernanponcedeleon** [comment]
Merged into `address-simplify` branch.
Closing this issue depends on #60.

**Comment by hernanponcedeleon** [comment]
Merged into master by a600260830801abe4cc9180040a1a20ee45a1672
